examples/demo_astradb.ipynb

```diff
@@ -0,0 +1,274 @@
+{
```

Comment:
<p>would be good to show a snippet from this document (e.g. documents[0].get_content()[10000:15000])) showing what the formatted text looks like, particularly around a section of this paper with tables/different sections </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_parse/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>